### DISCUSSION TITLE
I cannot build model-viewer in windows 11.

### DISCUSSION THREAD

**Discussion by Jayy0906** [google/model-viewer#4375] [Q&A]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: vX.X.X

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by diegoteran** [comment]
Hi @Jayy0906 
We won't be able to help with so little info. Please fill out the template completely, particularly the repro link, a screen recording, and any JS console errors/warnings.

**Comment by elalish** [comment]
At which of these steps did you encounter an error? https://github.com/google/model-viewer#windows-1011-setup
